### PR TITLE
Update hello_world.md

### PR DESCRIPTION
**Pull request by ahndmal**
delete wrong extra tick sign in CLI code

## Explanation

Correct the CLI cocde in docs.

## Related issue

No existing issue

## What type of PR is this

/kind documentation

## Proposed Changes

Correct the CLI cocde in docs.

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/develop/go/hello_world.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ $ wasmedge hello.wasm
 Hello TinyGo from WasmEdge!
 ```
 
-For the `--reactor` mode,
+For the `--reactor` mode:
```
Comment (adithyaakrishna) [inline docs/develop/go/hello_world.md:118]: @ahndmal Comma punctuation is well suited in this case. So could you please revert that one? The other change regarding extra backtick looks good to me

Comment (ahndmal) [reply docs/develop/go/hello_world.md]: Hello @adithyaakrishna !
Sure, changed it back to comma :) Thanks

**Review by adithyaakrishna — APPROVED** [review]
@hydai Could you please take another look at this PR :)

**Comment by hydai** [comment]
The DCO check failed. Please refer to this for signing DCO: https://www.secondstate.io/articles/dco/

**Review by hydai — CHANGES_REQUESTED** [review]
Nice catch. However, the DCO failed.

**Comment by adithyaakrishna** [comment]
@ahndmal Could you please sign-off your commits? More info on how to: https://github.com/WasmEdge/docs/pull/178/checks?check_run_id=17924620845

**Comment by ahndmal** [comment]
Signed the commits.
Please also note that https://www.secondstate.io/articles/dco/ page seems to have incorrect flags in:
``` git commit —amend —no-edit ---signoff ``` to be ```git commit --amend --no-edit --signoff ```

**Comment by hydai** [comment]
Please squash your commits. One of your commits is not DCO-signed.

**Comment by alabulei1** [comment]
Hi @ahndmal 

Could you please take a look at the DCO issue? It still fails. Thanks!

**Comment by hydai** [comment]
Due to the DCO not being signed, we are not able to merge this PR. Closing it for now. Please feel free to re-submit a new PR if the DCO is signed.

**Pull request closed without merging**